src/sentry/db/models/fields/array.py

```diff
@@ -69,3 +71,6 @@ def to_python(self, value):
                     assert "\\" not in value, "Unexpected ArrayField format"
                     value = value[1:-1].split(",")
         return [self.of.to_python(x) for x in value]
+
+
+DjangoArrayField.register_lookup(ArrayElementContainsLookup)
```

Comment:
I haven't found better/more fitting place to register this lookup.

Newly added lookup isn't registered to our custom `ArrayField` because our implementation of the field (`get_prep_value` [method](https://github.com/getsentry/sentry/blob/67db76651c2ff49d07acb309022bf2554e3d8b09/src/sentry/db/models/fields/array.py#L44-L54)) would require changes in order to work with this new lookup. I am not even sure if that is needed, but if it will be, that can be done as a follow up PR